oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -1015,7 +1018,7 @@ private String getConsoleUrl(String cluster, String actionId) throws FalconExcep
             // Get conf when workflow is launched.
             if (coordinatorAction.getExternalId() != null) {
                 WorkflowJob jobInfo = client.getJobInfo(coordinatorAction.getExternalId());
-
+                workflowUser = jobInfo.getUser();
```

Comment:
@bvellanki : Can you please verify what message.getWorkflowUser() is if falcon entity is scheduled as a different user than falcon?

If it's falcon then fix you provided should be moved here so that it can handle multiple WF engines : say FalconWorkflowEngine, OozieWorkflowEngine and any other engine supported in future.